apps/system/js/modal_dialog.js

```diff
@@ -362,23 +362,24 @@ var ModalDialog = {
 
   buildSelectOneDialog: function md_buildSelectOneDialog(data) {
     var elements = this.elements;
-    elements.selectOneTitle.textContent = data.title;
-    elements.selectOneMenu.innerHTML = '';
+    this._localizeElement(elements.selectOneTitle, data.title);
```

Comment:
I did not mean to write `{ raw: data.title }`, as I want to look up the properties by it's ID, which is the "default" behavior of passing just a string.

This is an API change for other users of `selectOne`. I believe I've now fixed the only other user of it

Also, I think I've cleaned up the events in the way you describe.